wiki/en/Running-a-Server.md

```diff
@@ -284,33 +253,39 @@ When the Server is running, the operating system will show an icon in the system
 
-The exact setup of port forwarding differs for every router. For help see [portforward.com](https://portforward.com).
+**Note:** The default port for the current version of Jamulus is **UDP** (not TCP) port **22124**. You will usually forward the port **22124** from outside your network to the port **22124** of the machine running the Server.
 
-Once you have your router set up, you can get your external (WAN) IP address e.g. by [using Google](https://www.google.com/search?q=what+is+my+ip). Give this address to your friends so they can connect to your Server (_but see also the note on dynamic DNS below_). You yourself should connect using the local network (LAN) address of the machine the Server is running on. If you are running a Client on the same machine as your Server, that would be `localhost` or `127.0.0.1`. 
+**Note:** Your home router may change the IP address of the machine you're running your Server on. Depending on your router, you might need to give this machine a static IP address (often under DHCP settings of your router).
 
-## Dynamic DNS and why you will probably need it
+#### Getting the external IP
 
-Most domestic Internet connections will change their IP address after a period (hours, days or weeks). To make it easier for people to connect to you, you may therefore also want set up a dynamic DNS address. You can do this on [the machine](https://www.online-tech-tips.com/computer-tips/ddns-dynamic-dns-service/) that you are running the Jamulus Server on, or preferably your router [may support it](https://www.noip.com/support/knowledgebase/how-to-configure-ddns-in-router/).
+To allow others to connect to your Server from the internet, get your external (WAN) IP address e.g. by [using Google](https://www.google.com/search?q=what+is+my+ip) and give it to them.
+You yourself should connect using the local network (LAN) address of the machine the Server is running on. If you are running a Client on the same machine as your Server, that would be `localhost` or `127.0.0.1`.
 
-Note also that your home router may also change the IP address of the machine that you are running your Server on. In which case you may need to give that machine a static IP in the router's DHCP configuration.
 
-# Backing up the Server
+#### Dynamic DNS and why you will probably need it
+
+Most domestic internet connections change their external IP address after a short period, you might want to set up a "dynamic DNS" which gives you a static (sub-)domain you can access and share with others. For further information, please look if your router already provides this feature or use an external DynamicDNS provider.
+
+## Backing up the Server
 
 _Note that headless Servers do not use `.ini` files. All configuration is given as command line options._
 
 {% include_relative Include-Backing-Up.md %}
 
-# Command line options
+## Command line options
```

Comment:
I'd move this section up before line 256 (new) "Running an Unregistered Server".

Reply:
Moved.